### PR TITLE
Bug 1942915: customizing-the-login-page: remove error template section

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @stlaz @bergerhoffer

### CODE REVIEW

File: modules/customizing-the-login-page.adoc [hunk 1]
```diff
@@ -28,11 +27,6 @@ $ oc adm create-login-template > login.html
 ----
 $ oc adm create-provider-selection-template > providers.html
 ----
-+
-[source,terminal]
-----
-$ oc adm create-error-template > errors.html
-----
```
Comment (stlaz) [inline modules/customizing-the-login-page.adoc:35]: why the removal of error-template-related commands?

Comment (s-urbaniak) [reply modules/customizing-the-login-page.adoc:35]: I may be wrong, but I don't see error page template renderers being used in the login page https://github.com/openshift/oauth-server/blob/822478f88514a80f053d9f65cd689d981b6ea4fd/pkg/oauthserver/auth.go#L354-L355 :thinking:

Comment (s-urbaniak) [reply modules/customizing-the-login-page.adoc:35]: as discussed OOB: templating still gets applied to the login page as it is being used by external IdPs to redirect to :+1: I suggest we simply enhance the description similar to how we did for the oc subcommand to create the error template.

**Comment by bergerhoffer** [comment]
@s-urbaniak Thanks for this. Can you tell me which OCP versions this applies for?

To make sure that I understand - was this saying that you could create custom error pages for when logins to the console fail, but you actually can't (which is why it was removed in this PR)?

**Comment by stlaz** [comment]
Looks good to me :+1: Not sure I'm supposed to `/lgtm` or have another docs member do that instead

**Pull request closed without merging**